keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+high-value experience for Kubernetes consumers.
+
+The Kubernetes community faces a reasonable amount of tech debt across
+infrastructure, testing, policy, and documentation. This KEP proposes that we
+spend more time paying down that debt.
+
+SIG Release currently produces releases at the following cadence:
+
+- patch releases (`x.y.Z`): [monthly][https://git.k8s.io/sig-release/releases/patch-releases.md]
+- minor releases (`x.Y.z`): [every four months][https://git.k8s.io/sig-release/release-engineering/versioning.md]
+- pre-releases (`x.y.0-(alpha|beta|rc).N`): every 1-3 weeks during active
+  development cycles ([example](https://git.k8s.io/sig-release/releases/release-1.21/README.md#timeline))
+
+At the time of writing, SIG Release considers these to be reasonable cadences
+for patch and pre-releases.
+
+If you'd like to provide suggestions on longer-term improvements that could
+potentially accelerate production of releases, please join the discussion
+[here](https://github.com/kubernetes/sig-release/discussions/1495).
+
+#### Establishing maintenance/stability releases
+
+Establishing a shorter maintenance/stability release at the end of the year has
+been casually discussed at several points in the project, with the most recent
+(at the time of writing) occurrence being
+[here](https://github.com/kubernetes/sig-release/issues/809).
+
+Nothing compelling has emerged from previous conversations to give cause to
+establish maintenance/stability releases.
+
+Fixing bugs, stabilizing components, adding/deflaking tests, improving
+documentation, and graduating features are activities that can and should
+happen in a reasonably consistent manner throughout the year.
+
+#### Determining an upper bound for Release Team shadows
+
+It was noted that fewer releases for the year would lead to fewer opportunities
+to participate on the Release Team.
+
+This will be discussed and addressed in
+https://github.com/kubernetes/sig-release/issues/1494.
+
+## Proposal
+
+### User Stories
+
+Kubernetes releases are made by real people. The technical aspects—for example,
+the release automation—reflects only a tiny part of the complete cycle. This
+means we will mainly focus on the human aspects and their corresponding roles
+when deciding to move to a 3-releases-per-year cadence.
+
+#### End User
+
+Most companies are facing issues upgrading Kubernetes 4 times a year. Providing
+only 3 releases per year will relax this situation.
+
+#### Distributors and downstream projects
+
+Downstream projects assemble their solution from different projects. Having
+fewer upgrades helps them to reduce complexity. For example, cloud providers
+will gain more room for upgrading their infrastructure.
+
+#### Contributors
+
+With a lower release cadence, contributors will gain more time for project
+enhancements, feature development, planning, and testing. It will provide more
+room for maintaining their mental health and prepare for events like KubeCon.
```

Comment:
that is very happy scenario, I doubt lot of people is 100% kubernetes focused, there is a big load of downstream work

Reply:
I know, I rephrased the sentence to:

> It will provide more room for maintaining their mental health, prepare for events like KubeCon or work on the downstream integration.